packages/slice-machine/src/features/onboarding/OnboardingGuide/OnboardingGuide.module.css

```diff
@@ -0,0 +1,23 @@
+.container {
+  overflow: hidden;
+  position: relative;
+  opacity: 1;
+  visibility: visible;
+  transition: opacity 500ms linear;
+}
+
+.confettiCannon {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.invisible {
+  opacity: 0;
+  visibility: hidden;
```

Comment:
Do we need both opacity and visibility?

Reply:
Yeah I guess the opacity is sufficient here 😅